api/datalake_api/app.py

```diff
@@ -37,10 +37,12 @@
 
 logging.getLogger('boto3.resources.action').setLevel(logging.WARN)
 
-sentry_dsn = app.config.get('SENTRY_DSN')
-if sentry_dsn is not None:
-    from raven.contrib.flask import Sentry
-    sentry = Sentry(app, dsn=sentry_dsn)
+try:
+    import sentry_sdk
+    from sentry_sdk.integrations.flask import FlaskIntegration
+    sentry_sdk.init(integrations=[FlaskIntegration()])
```

Comment:
Why do we need to be cautious about importing this? It seems to be in install requires...

Reply:
Yeah good point. Fixing....